cmd/envs.go

```diff
@@ -163,69 +153,92 @@ func listEnvsCmd(ctx *cli.Context) error {
 	client := api.NewClient(cfg)
 	c := context.Background()
 
-	// Look up the target org
 	var org *envelope.Org
-	org, err = client.Orgs.GetByName(c, ctx.String("org"))
-	if err != nil {
-		return errs.NewExitError(envListFailed)
-	}
-	if org == nil {
-		return errs.NewExitError("Org not found.")
+	var orgs []envelope.Org
+
+	// Retrieve the org name supplied via the --org flag.
+	// This flag is optional. If none was supplied, then
+	// orgFlagArgument will be set to "". In this case,
+	// prompt the user to select an org.
+	orgName := ctx.String("org")
```

Comment:
Could we dry this up? We repeat it in several different places.

Reply:
Done